Apps/phresilient/resilient.json

```diff
@@ -0,0 +1,10568 @@
+                "handle_format_is_name": {
+                    "description": "Treat handles as a name. Default is true",
+                    "data_type": "boolean",
+                    "default": true,
+                    "order": 0
+                },
+                "incident_id": {
+                    "description": "ID of incident",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "order": 1,
+                    "contains": [
+                        "ticketid"
+                    ]
+                },
+                "comment_id": {
+                    "description": "ID of comment to retrieve",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "order": 2,
+                    "contains": [
+                        "commentid"
+                    ]
+                },
+                "incidentcommentdto": {
+                    "description": "comment data as JSON String, format is IncidentCommentDTO data type from API",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 3
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.comment_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "109"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.handle_format_is_name",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.incident_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "2101"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.incidentcommentdto",
+                    "data_type": "string",
+                    "example_values": [
+                        "{\"text\":\"Comment updated\"}"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.comment_perms.delete",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    
```

Comment:
Questioning if this is correct `contains` although I guess its hard to tell as a community app